src/harness/renderers/gl/gl_renderer.c

```diff
@@ -549,6 +550,10 @@ void GLRenderer_Model(br_actor* actor, br_model* model, br_matrix34 model_matrix
         glUniform1i(uniforms_3d.light_value, -1);
     }
 
+    twosided = actor->material && actor->material->flags & BR_MATF_TWO_SIDED;
+    if (twosided)
+        glDisable(GL_CULL_FACE);
```

Comment:
I would put this code in SetActiveMaterial.

And by framing it like this, we don’t have to have a separate piece of reset code.
if (double sided)
  glDisable(culling)
else
  glEnable(culling

Reply:
I thought about this initially, but the problem with doing so is that you can leave the culling disabled if the last model had `BR_MATF_TWO_SIDED`. So if anything else is rendered in the frame after the models, it might cause problems.

Reply:
I think that’s ok - either the next thing to be rendered is
- another model with its own materials which will setup the rendering
config with its own settings, - that model was the last one, and we move on
to rendering the framebuffer which in this case doesn’t care about the
culling, but generally should be setting the rendering config regardless of
what the current config is

Reply:
Updated the patch to reflect the discussion.